manage_breast_screening/notifications/README.md

```diff
@@ -12,3 +12,49 @@ Storage is handled via Azure blob containers, Azure storage queues and Postgresq
 
 Appointment notifications are sent 4 weeks prior to the appointment date.
 Any appointment data processed within 4 weeks of the appointment date will also be eligible for notification on the next scheduled batch.
+
+## Running Tests
+
+### End-to-End Tests
+
+To run the end-to-end tests for notifications:
+
+```bash
+make test-end-to-end
+```
+
+This command will:
+
+- Start the required test dependencies (Azurite, MESH sandbox, and NHS Notify API stub)
+- Run the end-to-end test suite
+- Clean up the test environment afterwards
+
+The e2e tests are located in `manage_breast_screening/notifications/tests/end_to_end/` and test the complete notification flow from data ingestion through to message sending.
+
+**Prerequisites:**
+
+- Docker must be running
+- PostgreSQL database must be available (run `make db` first if needed)
+
+**Troubleshooting:**
+
+- If you get port conflicts, ensure no other MESH sandbox containers are running
+- If you get database connection errors, ensure PostgreSQL is running with `make db`
+
+### Integration Tests
+
+To run the integration tests:
+
+```bash
+make test-integration
+```
+
+### Unit Tests
+
+To run the notifications unit tests specifically:
+
+```bash
+poetry run pytest manage_breast_screening/notifications/tests/ --ignore manage_breast_screening/notifications/tests/dependencies --ignore manage_breast_screening/notifications/tests/integration --ignore manage_breast_screening/notifications/tests/end_to_end
```

Comment:
I think it's fine here to document the `make test-unit` command even if it is more of a general project level test run. It's also a dependency of the integration  tests in CI